products/extensions/b2b-components/employee-management/concept/README.md

```diff
@@ -0,0 +1,16 @@
+# Concept
+
+## Basic idea
+
+Employee management, as part of the B2B components, is a feature that allows you to manage employees and their permissions as an extension to Shopware's account and customer management, but gathered in a company context. This means that employees are associated with a **company customer** and will act on behalf of that company, when talking about e.g. placing orders in the shop using addresses and payment methods defined by administrators of such company.
+For that matter, we needed a way to provide order relevant data without building the checkout process anew and decided on using a **company customer**
+
+## Company customer
+
+The company customer is a regular customer in the shop, but with a few additional properties. This customer's ID is used to associate employees with a company. Therefore, it is really easy to relate on Shopware's typical order relevant data like addresses and payment methods.
+Internally, every employee will act like it is the company customer itself, using mostly the default handling of Shopware's Core.
+The entities of the B2B components will persist additional like the name of the employee in addition for internal purposes like the distinction of employee actions.
+
+## Role management
+
+Employees can be assigned roles that define their permissions and certain settings. These permissions can restrict or allow employees to perform certain actions, like ordering or managing roles or employees themselves. Learn how permissions can be extended [via plugin](products/extensions/b2b-components/employee-management/guides/creating-own-permissions-via-plugin.md) or [via app](products/extensions/b2b-components/employee-management/guides/creating-own-permissions-via-app.md) in our guides section.
```

Comment:
```suggestion
Roles define permissions and certain feature settings for employees. These permissions can restrict or allow certain actions, like placing orders or managing roles or employees themselves. It is supposed to provide an entrypoint for extensions, as B2B requirements heavily depend on your individual projects. 

Learn how permissions can be extended [via plugin](products/extensions/b2b-components/employee-management/guides/creating-own-permissions-via-plugin.md) or [via app](products/extensions/b2b-components/employee-management/guides/creating-own-permissions-via-app.md) in our guides section.
```

Reply:
We re-wrote this section. Do you think its fine now?